simrel.aggran

```diff
@@ -108,8 +108,8 @@
     <project
         name="Eclipse EMF"
         site="https://projects.eclipse.org/projects/modeling.emf.emf"
-        version="https://projects.eclipse.org/projects/modeling.emf.emf/releases/2.37.0"
-        releaseDate="2024-03-13"
+        version="https://projects.eclipse.org/projects/modeling.emf.emf/releases/1.38.0"
```

Comment:
> 1.38.0

Looks like a major degrade @merks

Reply:
Oops. I screwed creating my release record!

Reply:
Cool, I could just rename it:

https://projects.eclipse.org/projects/modeling.emf.emf/releases/2.38.0

Thanks for alerting me!  ❤️